src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  onSelectTodoCard: (todoCard: Todo) => void;
+  selectedTodoCard: Todo | null;
```

Comment:
just put to this component 
```suggestion
  isSelected: boolean;
```